README.md

```diff
@@ -0,0 +1,29 @@
+#  International Components for Unicode
+
+This is the repository for the [International Components for Unicode](https://ssl.icu-project.org). The ICU-TC is a technical committee of [The Unicode Consortium](https://unicode.org).
```

Comment:
The reference to the ICU-TC may be a little out of place at this level. It's not going to mean anything to anyone who doesn't know what the TC is already. Otherwise, looks good. Covers the most important links.

Reply:
Would it maybe be better to say that ICU is a project of the Unicode Consortium? (Not sure the wording is right though.)

Reply:
it could just say `The ICU project is under the stewardship of [The Unicode Consortium](https://unicode.org).`